policy_tests/run_unit_tests.py

```diff
@@ -105,13 +105,18 @@ def runTest(test, runtime, policy, pex, sim, rule_cache, rule_cache_size, output
     if exe_dir is not "tests":
         run_args += ["-S", exe_dir]
 
-    if timeout:
-        process = subprocess.Popen(["timeout",str(timeout),run_cmd] + run_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    else:
-        process = subprocess.Popen([run_cmd] + run_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    run_output,run_error = process.communicate()
-    if process.returncode != 0:
-        pytest.fail("Runtime failed with error: \n{}".format(run_error))
+    failed_msg = ""
+    try:
+        # Using PIPE for stdout sometimes prevents isp_run_app from noticing that the
+        # process has ended. Change to None for debugging
```

Comment:
Clean up this comment (unless the debugging hasn't yet been resolved)

Reply:
@JohnPMerrill The comment was to remind me in the future why I was sending stdout/stderr to /dev/null.

Would changing it to this help? If it's confusing either way, I can delete it. Now that I've spent time on it, it'll take me less time to figure out in the future.
```
        # Using PIPE for stdout sometimes prevents isp_run_app from noticing that the
        # process has ended. None won't hang, but will print `isp_run_app`s stdout.
        # DEVNULL hides the output and doesn't hang.
```

Reply:
Yes, that reads much clearer to me